docs/3.tutorials/auction/6-indexing.md

```diff
@@ -0,0 +1,85 @@
+---
+id: indexing-historical-data
+title: Indexing historical data
+sidebar_label: Indexing historical data
+---
+
+import {Github, Language} from "@site/src/components/codetabs"
+
+TODO: change github to main when merged
+
+In our frontend, we can easily display the previous bid since it's stored in the contract's state. However, we're unable to see previous bids to the auction. An indexer is used to fetch historical data from the blockchain and store it in a database. Since indexers can take a while to set up and can be expensive to run, we will use a pre-defined API point provided by NEAR Blocks to query an indexer they run that will fetch us the data we need.
+
+---
+
+## NEAR Blocks API key
+
+NEAR Blocks provides a free tier that allows you to make 6 calls per minute, this will be plenty for our usecase. To get an API key head over to https://dash.nearblocks.io/user/overview and sign up. Once signed go to `API Keys` then click `Add key` and give it whatever name you like. 
+
+We'll create a new file named `.env.local` to store our API key. Swap the API key in the example with your own.
+
+<Language value="bash" language="bash" showSingleFName={true}>
+    <Github fname=".env.local" 
+        url="https://github.com/PiVortex/auctions-tutorial/blob/main/frontend/.env.local"
```

Comment:
should this be `github.com/near-examples/...`?

Reply:
Yep I can change now, was waiting to be merged